variational_longitudinal_rate_of_aging_autoencoder.py

```diff
@@ -164,51 +164,32 @@ def fill_feed_dict_longitudinal(self,
         }
         return feed_dict
     
-    def get_longitudinal_loss(self):
-        # loss function for longitudinal data. 
-        
-        # initial position in the latent space (at timepoint 0) is just self.Z. 
-        Z0 = self.Z
-            
+    def get_Z1_from_Z0(self, Z0):
         # now project Z0 forward to get Z1. 
         # This requires multiplying the age components by longitudinal_ages1 / ages
         Z1 = tf.concat([Z0[:, :self.k_age] * tf.reshape((1.0*self.longitudinal_ages1 / self.ages), [-1, 1]), # broadcasting
                         Z0[:, self.k_age:]], 
                        axis=1)
+        return Z1
 
-        # reconstruct X from Z0 and Z1. 
-        Xr0 = self.decode(Z0)
-        Xr1 = self.decode(Z1)
-
-        # split the reconstructions up into binary and continuous components
-        Xr0_logits, Xr0_continuous = self.split_into_binary_and_continuous(Xr0) 
-        Xr1_logits, Xr1_continuous = self.split_into_binary_and_continuous(Xr1)
-
-        # do the same for the true data
-        X0_binary, X0_continuous = self.split_into_binary_and_continuous(self.X)
-        X1_binary, X1_continuous = self.split_into_binary_and_continuous(self.longitudinal_X1)
-
-        # compute losses
-        binary_loss_0 = self.get_binary_loss(X0_binary, Xr0_logits)
-        continuous_loss_0 = self.get_continuous_loss(X0_continuous, Xr0_continuous)
-        binary_loss_1 = self.get_binary_loss(X1_binary, Xr1_logits)
-        continuous_loss_1 = self.get_continuous_loss(X1_continuous, Xr1_continuous)
+    def get_longitudinal_loss(self, 
+                              binary_loss0, 
+                              continuous_loss0, 
+                              binary_loss1,
+                              continuous_loss1,
+                              reg_loss):
 
```

Comment:
do you think the variable names in variational_longitudinal_rate_of_aging_autoencoder make sense? (longitudinal_X1 etc.) i was worried when i chose them they were confusing.